### DISCUSSION TITLE
Getting 404 for everything

### DISCUSSION THREAD

**Discussion by dzecevic** [anancarv/python-artifactory#141] [Q&A] [answered]
Hi!

What am I doing wrong?
We have Artifactory PRO version  7.49.3 and however i try to get some data i got 404

What code I have:

```
from pyartifactory import Artifactory

ARTIFACTORY_URL="https://artifactory-test.SOMETHING.net"
USERNAME="*"
PASSWORD_OR_API_KEY="*!"

art = Artifactory(url=ARTIFACTORY_URL, auth=(USERNAME,PASSWORD_OR_API_KEY), verify=False, api_version=1)

groups = art.groups.list()

print (groups)
```

I have tried with api_version=1 and api_verison=2

And everytime i got
`requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://artifactory-test.SOMETHING.net/api/security/groups`

And as i read in artifactory documentation (https://jfrog.com/help/r/jfrog-rest-apis/get-a-list-of-groups) api should be: /access/api/v2/groups

**Comment by nymous** [comment, marked as answer]
Hello!
Can you try to specify your Artifactory URL with the `/artifactory` at the end? like `https://artifactory-test.SOMETHING.net/artifactory`

Also you should know that the library was written for the "old" Security APIs, when they were exposed on the artifactory service instead of the access service (https://jfrog.com/help/r/jfrog-rest-apis/artifactory-security).

**Comment by dzecevic** [comment]
Thanks! It works!
